Documentation/development-environment.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Multi-Node Test Environment
+weight: 91
+indent: true
+---
+
+# Rook development workflow
+
+## Setup expectation
+
+There are a bunch of pre-requisites to be able to deploy the following environment. Such as:
+
+* A Linux workstation (CentOS or Fedora)
+* KVM/QEMU installation
+
+For other Linux distribution, there is no guarantee the following will work.
+However adapting commands (apt/yum/dnf) could just work.
+
+## Prerequisites installation
+
+Execute `tests/scripts/multi-node/redhat-system-prerequisites.sh`.
+
+## Deploy Kubernetes with Kubespray
+
+In order to successfully deploy Kubernetes with Kubespray, you must have this code: https://github.com/kubernetes-incubator/kubespray/pull/2153 and https://github.com/kubernetes-incubator/kubespray/pull/2271.
```

Comment:
these two PRs are already merged. Can we just suggest to use the latest kubespray?